src/v/kafka/server/group.cc

```diff
@@ -3143,18 +3147,18 @@ ss::future<> group::do_abort_old_txes() {
         // if no error was triggered during abort of transaction we may try to
         // schedule a next expiration earlier if there are transactions pending
         // to be expired
-    maybe_rearm_timer();
-}
+        maybe_rearm_timer();
+    }
```

Comment:
nit: this change belongs in the earlier commit?

Reply:
done